cranelift/codegen/src/isa/riscv64/lower.isle

```diff
@@ -697,63 +697,63 @@
 
 ;;;;;  Rules for `uload8`;;;;;;;;;
 (rule
-  (lower (uload8 flags p offset))
+  (lower (uload8 flags p @ (value_type (ty_64 _)) offset))
```

Comment:
Does `ty_64` succeed for `f64`? I guess the CLIF verifier would presumably catch that one, but it may make sense to use a purpose built extractor that is called `is_address_type` or something like that.

Reply:
It looks like the instruction definition (the `iAddr` type set) includes only ints and reference types, so this case can't happen. I do like `is_address_type` though; I'll do that.